### DISCUSSION TITLE
Migrated from Webpacker to Vite Ruby

### DISCUSSION THREAD

**Discussion by BilalBudhani** [ElMassimo/vite_ruby#92] [Show and tell]
Hi,

I just wanted to thank the creator and contributors of this gem. I just migrated a small project (Inertia + Vue 2) from Webpacker to Vite Ruby and everything went smoothly. The documentation and error messages were a big help too. 

A big thank you to everyone behind for their efforts. 🙏

**Comment by ElMassimo** [comment]
Hi Bilal!

Thanks for sharing, happy to hear that! 😃

Let me know if you have any suggestions on error messages and documentation, so that I can keep improving this library.

If you are using Inertia you might want to check out [JS From Routes](https://github.com/ElMassimo/js_from_routes), which can simplify how you [make requests](https://github.com/ElMassimo/pingcrm-vite/pull/4) to Rails.

Cheers!

**Reply by BilalBudhani** [reply]
Yes, I'm gonna try out JS From Routes gem as well, I saw you've Inertia plugin for that, which is awesome. I'm currently using https://github.com/railsware/js-routes but JS From Routes generates cleaner code.
